library/db/src/main/java/org/keynote/godtools/android/db/GodToolsDao.kt

```diff
@@ -87,7 +87,7 @@ class GodToolsDao @Inject internal constructor(database: GodToolsDatabase) : Abs
 
     public override fun getPrimaryKeyWhere(obj: Any) = when (obj) {
         is LocalFile -> getPrimaryKeyWhere(LocalFile::class.java, obj.filename)
-        is TranslationFile -> getPrimaryKeyWhere(TranslationFile::class.java, obj.translationId, obj.filename!!)
+        is TranslationFile -> getPrimaryKeyWhere(TranslationFile::class.java, obj.translationId, obj.filename)
```

Comment:
What did the ```!!``` do that we no longer need?

Reply:
`!!` forces a value to be non-null, or it will throw an exception if it was null. now that `TranslationFile` can no longer have a null `filename`, there is no reason to force it to be non-null here.